internal/completions/types/types.go

```
@@ -58,6 +59,7 @@ type CompletionRequestParameters struct {
 	TopK              int       `json:"topK,omitempty"`
 	TopP              float32   `json:"topP,omitempty"`
 	Model             string    `json:"model,omitempty"`
+	Stream            bool      `json:"stream,omitempty"`
```

This struct is shared among chat and code completions, so the property will actually have no effect for chat rn. Although it has no effect, it's value will be `false` by default, which I think can be misleading.